CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.0-beta.8
```

Comment:
Please also update the version number in the README. This should probably be a pending version, since there are a couple other changes I'd like to land.